spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/propagation/DefaultBasePropagationFactoryProvider.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2013-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.sleuth.propagation;
+
+import brave.propagation.B3Propagation;
+import brave.propagation.Propagation;
+
+/**
+ * Provides the default {@link brave.propagation.Propagation.Factory}
+ *
+ * @author Brian Devins-Suresh
+ * @since 2.1.0
+ */
+public class DefaultBasePropagationFactoryProvider implements BasePropagationFactoryProvider {
```

Comment:
Maybe `B3PropagationFactoryProvider` ? Also you can make it package scope

Reply:
Removed